roles/openshift_control_plane/tasks/systemd_units.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install master services files
+  template:
+    src: "{{ item }}.j2"
+    dest: "/etc/systemd/system/{{ openshift_service_type }}-{{ item }}"
+    owner: root
+    group: root
+    mode: 600
+  with_items:
+  - master-api.service
+  - master-controllers.service
+  - master-etcd.service
+  notify:
+  - reload systemd units
```

Comment:
need to add this handler back.  i think i thought it was a task before, not a handler.  last thing as long as nothing else changes.